content/blog/tyranny-of-nits/index.md

```diff
@@ -0,0 +1,208 @@
+And even [very protracted discussions](https://github.com/bevyengine/rfcs/pull/45) can lead to a much better solution, so concerns *should* be raised!
+
+But interminable debate is a huge risk factor for burnout, which often leads to no solution at all, and can even make trying to tackle the problem again radioactive.
+Your objections can be as civil, clear and reasonable as you'd like: keep at it long enough and both the author and everyone who wants this feature will hate you.
+
+So how do we keep these essential conflicts healthy for everyone involved.
+I think there's three key elements:
+
+1. Have a process lifecycle for important changes, and don't regress without extremely good reason.
+2. Talk clearly about the consequences of each proposal, and how hard it will be to fix mistakes.
+3. Cut debate short, and actually decide things.
+
+Considering, designing and implementing controversial changes should be staged ([Niko's proposal is great!](https://smallcultfollowing.com/babysteps/blog/2023/09/18/stability-without-stressing-the-out/)).
+It doesn't really matter whether this is formal or informal (just make it clear to contributors).
+You need to answer the following questions:
+
+1. Should we do this?
+2. How do we do this?
+3. Are the details of this solution good enough?
+
+The first question is often the most important: every library and language has an infinite amount of things it *could* do, and the set of things it *maybe should* do is only smaller in a way that only a mathematician could love.
+Answering the question of "should we do this" is in my opinion the most important goal of things like [RFC processes](https://rust-lang.github.io/rfcs/).
+Build some consensus, come to a decision, and hopefully prioritize your work so that you don't have too many open "we should do this!" ideas floating around at once.
+
+The next stage is where all of the big architectural discussions should take place.
+Hash it out on paper, maybe even build some prototypes if needed to test some theories.
+Then come together and build something, even if your preferred design didn't win.
+
+Finally, it comes to evaluating the final implementation.
+In Rust, this tends to be the stabiliation process, but in most projects, this is just PR review.
+Here, you should be focusing on clear bugs and polish problems: compromise conflicts are already hashed out and future work has already been written up as you work.
+
+The key thing to learn here is shouldn't go backwards in that process without *very* good reason, especially when a lot of effort has been invested at previous stages: it will waste time, threaten projects and burn people out like a sparkler.
+Don't, by way of hypothetical example, pick a fight in on a stabilization report (stage 3), after years of implementation work, to argue about whether the feature should exist at all (a stage 1 concern).
```

Comment:
Is this a subtweet? It feels like a subtweet.

Reply:
Yes lol: this is about GATs.